ext/opentelemetry-ext-mysql/setup.cfg

```
@@ -41,8 +41,8 @@ package_dir=
 packages=find_namespace:
 install_requires =
     opentelemetry-api == 0.7.dev0
+    opentelemetry-ext-dbapi >= 0.7.dev0
```

```suggestion
    opentelemetry-ext-dbapi == 0.7.dev0
```

I don't remember the PR anymore but we decided to pin all the versions around for the time being.